features/step_definitions/user_steps.rb

```diff
@@ -45,6 +45,11 @@
 
 Given /^I am logged in as "([^\"]*)" with password "([^\"]*)"$/ do |login, password|
   step("I am logged out")
+  step(%{I am logged in as "#{login}" with password "#{password}" no logout step})
```

Comment:
Sorry, I meant why do you need a version which doesn't log out? Is it another JavaScript testing issue or something else? (Refactoring is good though!)